File: SS1/Ex8/CPP/Ex8.cpp
#include <iostream>
#include <fstream>
#include <string>
#include <sstream>
using namespace std;

template <class T>
string to_string(T t, ios_base & (*f)(ios_base&))
{
  ostringstream oss;
  oss << f << t;
  return oss.str();
}

int main ()
{
    int number;
    cout<<"Nhap so nguyen chuyen sang Hex: ";
    cin>>number;
    cout<<to_string<long>(number, hex)<<endl;
    system("PAUSE");
    return 0;
}

File: SS1/Ex8/Java/Ex8.java
// import java.util.*;
import java.util.Scanner;

public class Ex8{
  public static void main(String args[]) {
    int num;
    Scanner sc = new Scanner(System.in);

    System.out.print("Nhap mot so nguyen: ");
    num = sc.nextInt();

    String hexVal = "";
    hexVal = Integer.toHexString(num);
    System.out.println("Chuyen sang ma Hex: " + hexVal);
  }
}